package/yast2.changes

```diff
@@ -1,4 +1,17 @@
 -------------------------------------------------------------------
+Wed Nov  5 09:33:08 UTC 2014 - mfilka@suse.com
+
+- bnc#900383
+  - ported several fixes for device renaming
+  - added net device type detection based on sysfs
+  - fixed type detection workflow
+  - fixed device type detection when commiting new device into
+    NetworkInterfaces' cache.
+  - changes API for device type detection - incompatible to previous
+    versions
```

Comment:
This is the biggest potential problem. How do you ensure that yast2-network, which contains 1 reference to the old API, does not break?